src/extension.js

```diff
@@ -25,9 +26,30 @@ function activate(context) {
 	const BackupFilePath = uuid =>
 		path.join(base, "electron-sandbox", "workbench", `workbench.${uuid}.bak-custom-css`);
 
+	function resolveVariable(key) {
+		const variables = {
+			cwd: () => process.cwd(),
+			userHome: () => os.homedir(),
+			execPath: () => process.env.VSCODE_EXEC_PATH ?? process.execPath,
+			pathSeparator: () => path.sep,
+			"/": () => path.sep,
+		};
+
+		if (key in variables) return variables[key]();
+
+		if (key.startsWith('env:')) {
+			const [_, envKey, optionalDefault] = key.split(':');
+			return process.env[envKey] ?? optionalDefault;
+		}

```

Comment:
Add handling code if none of the options match -- throw an error or return empty

Reply:
What if the `key` doesn't start with `env:`?

Reply:
The function returns either a string (the resolved value of whatever we're trying to substitute) or undefined (meaning that it could not be resolved). 

In line 50,
(`.replaceAll(/\$\{([^\{\}]+)\}/g, (substr, key) => resolveVariable(key) ?? substr);`), if the return value of `resolveVariable` is undefined, it defaults to the original string (replace `substr` with `substr`), which is effectively a no-op.